### PR TITLE
Ekf2 publish ctrl states

### PR DESCRIPTION
**Pull request by tumbili**
@sverling Could you have a quick look at this. The acceleration data was still missing in the control state topic. This is important for the land detector to work well.
Also I think the rotation matrix you were using was transforming from body to earth frame.

### CODE REVIEW

**Comment by sverling** [comment]
sure. i will check probably today in the afternoon

File: src/modules/ekf2/ekf2_main.cpp [hunk 1]
```diff
@@ -489,11 +491,45 @@ void Ekf2::task_main()
...
+
+			// Attitude quaternion
 			ctrl_state.q[0] = q(0);
 			ctrl_state.q[1] = q(1);
 			ctrl_state.q[2] = q(2);
 			ctrl_state.q[3] = q(3);
 
+			// Acceleration data
+			matrix::Vector<float, 3> acceleration = {&sensors.accelerometer_integral_m_s[0]};
+			acceleration = acceleration / sensors.accelerometer_integral_dt[0];
```
Comment (sverling) [inline src/modules/ekf2/ekf2_main.cpp]: why don't we directly take sensors.accelerometer_m_s2?

**Comment by sverling** [comment]
Sorry for the late response. the changes look good to me, only question is the commend above.

File: src/modules/ekf2/ekf2_main.cpp (hunk 1 above)
Comment (LorenzMeier) [inline src/modules/ekf2/ekf2_main.cpp]: I think for the control state using the low passed accel data makes more sense. @tumbili Can you rebase change that? Thanks!

**Comment by tumbili** [comment]
Merged

**Pull request closed without merging**